### PR TITLE
Add Weaviate APIs to Gorilla API Zoo

### PR DESCRIPTION
**Pull request by CShorten**
# Add Weaviate APIs to Gorilla API Zoo

Following the template from the Google Calendar API example -- this sets `weaviate-client` as a requirement for the env and then assumes you have an aliased `weaviate_client` for accessing APIs with e.g. `weaviate_client.collections.get()...`.

### CODE REVIEW

File: data/apizoo/Weaviate_API.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+[
+  {
+    "user_name": "weaviate",
```
Comment (ShishirPatil) [inline data/apizoo/Weaviate_API.json:3]: should be GitHub handle "CShorten" to track provenance.

**Review by ShishirPatil — APPROVED** [review]
Thanks for the PR @CShorten ! Just a minor change listed above and also [this](https://github.com/ShishirPatil/gorilla/actions/runs/11999902809/job/33458623610?pr=783#step:4:36), and the rest looks good!

**Comment by CShorten** [comment]
Hey @ShishirPatil @CharlieJCJ, 

I think this will fix it, could we please rerun the workflow?

**Review by CharlieJCJ — APPROVED** [review]
LGTM, thanks @CShorten !

**Pull request merged**